src/datasources/asset/AssetDataSource.ts

```diff
@@ -70,6 +82,16 @@ export class AssetDataSource extends DataSourceBase<AssetQuery> {
     return result;
   }
 
+  async processCalibrationForecastQuery(query: AssetCalibrationForecastQuery, options: DataQueryRequest) {
+    const result: DataFrameDTO = { refId: query.refId, fields: [] };
+    const from = options.range!.from.toISOString();
```

Comment:
@kkerezsi what is the backend expecting here? I remember we discussed that we need to compute forecast in local time to place the calibration in the right bucket.

Reply:
@CiprianAnton We discussed with Josh and we agreed that it's not worth including the local time conversion since 2 users might view different results and it would only complicate the flow. I'd say we keep it simple and have consistency across executions.